### DISCUSSION TITLE
Why was the crowbar replaced?

### DISCUSSION THREAD

**Discussion by acayirli** [sultim-t/xash-rt#100] [General]
The mod stays very true to Half-Life, even restoring the original retail main menu to some degree, but for some reason they changed the crowbar model and animation. Why? Is it the crowbar from Black Mesa? It seems like it, and I think that's the worst crowbar implementation ever.

Well another departure from the original are the chapter titles, but I really like those tbh.

**Comment by saarstar15** [comment]
You can use the original instead by deleting the file: v_crowbar.mdl located in the following path:

hl1-rt-resources > rt > valve > models
